lib/blocks.php

```diff
@@ -40,8 +42,10 @@ function register_block_type( $name, $args = array() ) {
  *                                   complete WP_Block_Type instance.
  * @return WP_Block_Type|false The unregistered block type on success, or false on failure.
  */
-function unregister_block_type( $name ) {
-	return WP_Block_Type_Registry::get_instance()->unregister( $name );
+if ( ! function_exists( 'register_block_type' ) ) {
```

Comment:
Why not `if ( ! function_exists( 'unregister_block_type' ) )`?

Reply:
That was a typo, thanks for noticing.